cmd/crossplane/core/core.go

```diff
@@ -59,7 +62,15 @@ func (c *Command) Run(log logging.Logger) error {
 		return errors.Wrap(err, "Cannot get config")
 	}
 
-	mgr, err := ctrl.NewManager(cfg, ctrl.Options{SyncPeriod: &c.Sync})
+	mgr, err := controller.NewManager(cfg, controller.Options{
+		Options: &ctrl.Options{
```

Comment:
@negz I've already updated this here based on the corresponding `crossplane-runtime` PR. Although at this current state of the PR nothing will even get compiled, but this was for the sake of demonstration of the idea.